Makefile-docker

```diff
@@ -64,6 +64,12 @@ jquery-ui/ui/widgets/autocomplete.js \
 jquery-ui/ui/widgets/datepicker.js \
 jquery-ui/ui/widgets/sortable.js
 
+FLAKE8 := -r requirements/flake8.txt
+ifeq ($(PYTHON_VERSION_MAJOR), 2)
+	FLAKE8 += -r requirements/flake8_py2.txt
+endif
+
```

Comment:
I don't hate this way of doing it, but it's inconsistent with the `prod_py2.txt` and `prod_py3.txt` dep installs.

Reply:
you can just do an include at the top of the `flake8_py2.txt` file that includes the other file.

Reply:
I addressed this via c35ee83 and mirrored how we handle other _py3 dependency files.